src/conn/conn_sweep.c

```diff
@@ -279,11 +275,11 @@ __sweep_server(void *arg)
     for (;;) {
         /* Wait until the next event. */
         if (FLD_ISSET(conn->timing_stress_flags, WT_TIMING_STRESS_AGGRESSIVE_SWEEP))
-            __wt_cond_wait_signal(session, conn->sweep_cond,
-              conn->sweep_interval * 100 * WT_THOUSAND, __sweep_server_run_chk, &cv_signalled);
+            sweep_interval = conn->sweep_interval / 10;
```

Comment:
Not re-reading the value would cause the condition line 298 to be wrong:
```
        if (!cv_signalled && (now - last < sweep_interval))
```
The `sweep_interval` is read once, before we get into that loop.